presto-docs/src/main/sphinx/sql/create-table.rst

```diff
@@ -61,12 +61,13 @@ Create a new table ``orders``::
     )
     WITH (format = 'ORC')
 
-Create the table ``orders`` if it does not already exist, adding a table comment
-and a column comment and a primary key constraint on column ``orderkey``::
+Create the table ``orders`` if it does not already exist, adding a table comment,
+a column comment, a not null constraint on column ``orderstatus`` and a primary
```

Comment:
```suggestion
a column comment, a ``not null`` constraint on column ``orderstatus``, and a primary
```
Same formatting suggestion, also use of the serial (Oxford) comma following the [Gitlab documentation style guide Punctuation topic](https://docs.gitlab.com/ee/development/documentation/styleguide/#punctuation).